transport/grpc/server.go

```diff
@@ -72,6 +72,13 @@ func ServerErrorLogger(logger log.Logger) ServerOption {
 
 // ServeGRPC implements the Handler interface.
 func (s Server) ServeGRPC(ctx oldcontext.Context, req interface{}) (oldcontext.Context, interface{}, error) {
+
+	defer func() {
+		if r := recover(); r != nil {
+			s.logger.Log("panic", r)
+		}
+	}()
```

Comment:
It's important that this kind of functionality is opt-in. By default, I don't want my ServeGRPC (or ServeHTTP) to be catching, logging, and squelching panics.